tests/manage/pv_services/test_pvc_delete_subvolumegroup.py

```diff
@@ -30,13 +31,30 @@ class TestPvcDeleteSubVolumeGroup(ManageTest):
     subvolumegroup_ls_cmd = None
 
     @pytest.fixture(autouse=True)
-    def setup(self, project_factory):
+    def setup(self, request, project_factory):
         """
         create resources for the test
 
         Args:
             project_factory: A fixture to create new project
         """
+        if registry.check_if_registry_stack_exists():
+            logger.info("Removing OCP registry from ODF")
+            remove_ocp_registry_from_ocs(config.ENV_DATA["platform"])
+            registry_pvc = get_all_pvc_objs(
+                namespace=constants.OPENSHIFT_IMAGE_REGISTRY_NAMESPACE
+            )[0]
+            registry_pvc.delete()
+            registry_pvc.ocp.wait_for_delete(registry_pvc.name)
+            registry_removed = True
```

Comment:
If the script fails at any line of code between 44 to 48, the 'registry_removed' flag will not be set to true, and the registry will be left unconfigured in the finalizer